h/static/scripts/controllers/search-bar-controller.js

```diff
@@ -236,10 +243,19 @@ class SearchBarController extends Controller {
      */
     const addLozenge = (content) => {
 
+      const lozengeEl = cloneTemplate(this.options.lozengeTemplate);
```

Comment:
This stamps out a new lozenge from the template (as if it were server-rendered), insert it after the previous lozenge and then upgrade it.